src/olympia/users/admin.py

```diff
@@ -465,7 +465,9 @@ def last_known_activity_time(self, obj):
         # We sort by -created by default, so first() gives us the last one, or
         # None.
         user_log = (
```

Comment:
Nit: Because we no longer have such a thing as a `UserLog`, should we rename this variable to just `log` or `activity_log`?